site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md

```diff
@@ -244,6 +222,29 @@ meant to facilitate downgrading or reverting the option change. When the option
 is set to `false`, all database objects with extended names that were created
 in the meantime should be removed manually.
 
+### Changed TTL index removal default
+
+The default value of the `--ttl.max-collection-removes` startup option has been
+lowered from 1 million to 100,000. The background thread for time-to-live indexes
+now removes fewer documents from a collection in each iteration to give other
+collections a chance of being cleaned up as well.
+
+### In-memory cache subsystem
+
+By default, the in-memory cache subsystem uses up to 95% of its configured
+memory limit value (as configured by the `--cache.size` startup option).
+
```

Comment:
This not true anymore after we reverted the default value change.
So we should probably remove the entire section here, as it provides not much value if the default value remains as it was.